src/app/business/common/components/detail/components/metadata.js

```diff
@@ -160,6 +169,27 @@ OwnerMetadata.defaultProps = {
     owner: '',
 };
 
+export const MetadataMetadata = ({metadata}) => {
+    const keys = Object.keys(metadata);
```

Comment:
I think you're missing sorted() here as well :)

Reply:
Yes indeed!